.flox/env.json

```diff
@@ -0,0 +1,4 @@
+{
+  "name": "ort-flox",
+  "version": 1
+}
```

Comment:
What about the missing line break, or is this a generated file?

Reply:
It's a generated file.